1-exercises/A-this/exercise-1.js

```diff
@@ -9,18 +9,17 @@
 
 */
 
-
-
 let person = {
-    name: "Adrian",
-    age: 32,
-    currentAddress: "London",
-    celebrateBirthday: /* todo */
+  name: "Adrian",
+  age: 32,
+  currentAddress: "London",
+  celebrateBirthday: function () {
+    this.age++;
+  },
 };
 
-
 // DO NOT MODIFY BELOW THIS LINE
 let age = person.age;
```

Comment:
Really nice Laleh! The only thing I would say is I always encourage you to use `const` over `let` if we are not reassigning variables. You can also do `const { age } = person` to destructure objects :)

Reply:
Thanks a lot Michael ,Sure! thanks for letting me know about another way for destructuring objects